### ISSUE TITLE
Lost connection with local raylet. Error: IOError: 14: Socket closed

### ISSUE THREAD

**Issue by YarShev** [ray-project/ray#8129]
I see the problem when I use `modin` with `ray` backend. I installed it as `pip install modin` and `pip install modin[ray]`. My workflow is the following:

```bash
export MODIN_ENGINE=ray
export MODIN_BACKEND=pandas
```

```python
import modin.pandas as pd
df = pd.read_csv("path_to_my_csv")
```
I get

```bash
F0422 09:38:54.914767 17143 direct_task_transport.cc:154] Lost connection with local raylet. Error: IOError: 14: Socket closed
*** Check failure stack trace: ***
    @     0x7ff916a6098d  google::LogMessage::Fail()
    @     0x7ff916a61dfc  google::LogMessage::SendToLog()
    @     0x7ff916a60669  google::LogMessage::Flush()
    @     0x7ff916a60881  google::LogMessage::~LogMessage()
    @     0x7ff9168350e9  ray::RayLog::~RayLog()
    @     0x7ff9167560c9  ray::CoreWorkerDirectTaskSubmitter::RetryLeaseRequest()
    @     0x7ff916758863  _ZNSt17_Function_handlerIFvRKN3ray6StatusERKNS0_3rpc16WorkerLeaseReplyEEZNS0_29CoreWorkerDirectTaskSubmitter24RequestNewWorkerIfNeededERKSt4pairIiSt6vectorINS0_8ObjectIDESaISC_EEEPKNS4_7AddressEEUlS3_S7_E_E9_M_invokeERKSt9_Any_dataS3_S7_
    @     0x7ff91678fc3f  ray::rpc::ClientCallImpl<>::OnReplyReceived()
    @     0x7ff9167329d3  _ZN5boost4asio6detail18completion_handlerIZN3ray3rpc17ClientCallManager29PollEventsFromCompletionQueueEiEUlvE_E11do_completeEPvPNS1_19scheduler_operationERKNS_6system10error_codeEm
    @     0x7ff9167313a5  boost::asio::detail::scheduler::run()
    @     0x7ff9167354a3  ray::CoreWorker::RunIOService()
    @     0x7ff917fe5421  execute_native_thread_routine_compat
    @     0x7ff96974adc5  start_thread
    @     0x7ff9694781cd  __clone
Aborted
```
But this problem is resolved by setting `https_proxy` as empty.

**Comment by YarShev** [comment]
I think the problem above is related with this [issue](https://github.com/ray-project/ray/issues/6662)

**Comment by dvirginz** [comment]
What is the https_proxy everyone is talking about?:)

**Comment by YarShev** [comment]
Problem was resolved, I was using ray 0.8.0 where there was the problem. Now I am using 0.8.4 and it is ok.

**Comment by YarShev** [comment]
That is variable which is used by us to set proxy:)
 
By the way, the problem was resolved. I was using ray==0.8.0, but I saw that it was fixed in 0.8.3 release. Now I am using ray 0.8.4 and it is ok. I have closed the issue.
  
>Среда, 22 апреля 2020, 21:56 +03:00 от Dvir Ginzburg <notifications@github.com>:
> 
> 
>What is the https_proxy everyone is talking about?:)
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
 
--
Ярослав Игошев
